### DISCUSSION TITLE
Basic Github question

### DISCUSSION THREAD

**Discussion by m-holger** [m-holger/pdfarranger#8] [Q&A]
@dreua 

I noticed you signed off your moderation edit of issue 427 as `/@dauer` (did you mean `/@dreua` ?). What is the significance of the `/`?

1 is that a way to give credit to a contributer without generate a notification to him?
2 If so, does that work with mentions of issues, i.s. would `/#427` generate a link here without generating a back link/mention in 427?

BTW, I accidentally deleted the 'About box' discussion'. Oops. I did see all your replies. Thanks

**Comment by dreua** [comment]
Good catch, I meant @dreua of course. @dauer took my preferred username that I use as email and on other platforms, got that confused myself. (@dauer if you are reading this: Sorry for tagging you by mistake!)

The slash has no special meaning on GH afaik. I did this just from a gut feeling, I think it is a convention on some Facebook pages with a moderation team. (Maybe specific to Germany, not sure...)
I think it's important to clearly state who authored the moderation edit because it is not easily visible, maybe you have better suggestions than a simple slash?

**Reply by dreua** [reply]
So no to questions 1 and 2, or at least not that I'm aware of.

Just thinking about the slash some more: I think Wikipedia uses two dashes for signing comments, would that be better? --@dreua

Loosing the About Box discussion is unfortunate, I think it could have become valuable information for other new contributors. Is there any way to restore it or do you have a copy that you could mail me? I could give it a quick edit and put it in a wiki page for preservation.

**Reply by dreua** [reply]
Juste got a notification for mentioning /@dreua myself in an edit, so no, it does not suppress that for sure.

**Comment by m-holger** [comment]
On 27/03/2021 21:01, David Auer wrote:
>
> Just thinking about the slash some more: I think Wikipedia uses two 
> dashes for signing comments, would that be better? --'@'dreua
> <https://github.com/dreua>
>
In terms of identifying the author - I think either way is clear

In terms of making readers wonder whether there is something clever 
being done - the '--' probably would not have raised that question in my 
mind (the '/', even though its a forward '/', felt more like it might be 
an escape)
